src/ucs/datastruct/ptr_array.c

```diff
@@ -20,23 +20,18 @@
 #define UCS_PTR_ARRAY_INITIAL_SIZE  8
 
 
-static inline int ucs_ptr_array_is_free(ucs_ptr_array_t *ptr_array, unsigned idx)
+static inline int
+ucs_ptr_array_is_free(ucs_ptr_array_t *ptr_array, unsigned element_index)
 {
-    return (idx < ptr_array->size) &&
-            __ucs_ptr_array_is_free(ptr_array->start[idx]);
+    return (element_index < ptr_array->size) &&
+            __ucs_ptr_array_is_free(ptr_array->start[element_index]);
```

Comment:
minor: it seems that excessive whitespace here, could you check pls?

Reply:
?

Reply:
Well, I updated the code but I'm not sure what you mean by excessive whitespace on this line...